packages/core/ui/ResizeHandle.tsx

```diff
@@ -57,40 +69,33 @@ function ResizeHandle({ style, onDrag, vertical, flexbox }) {
     return cleanup
   }, [mouseDragging, onDrag, vertical])
 
-  function mouseDown(event) {
+  function mouseDown(event: React.MouseEvent) {
     event.preventDefault()
     const pos = event[vertical ? 'clientX' : 'clientY']
     prevPos.current = pos
     setMouseDragging(true)
   }
 
-  function mouseLeave(event) {
+  function mouseLeave(event: React.MouseEvent) {
     event.preventDefault()
   }
 
+  let className
+  if (flexbox) {
+    if (vertical) className = classes.flexbox_verticalHandle
+    else className = classes.flexbox_horizontalHandle
+  } else if (vertical) className = classes.verticalHandle
+  else className = classes.horizontalHandle
+
   return (
     <div
       onMouseDown={mouseDown}
       onMouseLeave={mouseLeave}
       role="presentation"
-      className={
-        classes[
-          (flexbox ? 'flexbox_' : '') +
-            (vertical ? 'verticalHandle' : 'horizontalHandle')
-        ]
-      }
-      style={style}
+      className={className}
+      {...props}
```

Comment:
what are the other props in this case?

Reply:
The styles are the only thing that I know get used in the other props. One of my iterations had an event handler on the resize handle, too, and even though I didn't end up going that way, I thought it might be useful in the future.